content/test/integration/controller/deployment-pagination.spec.ts

```diff
@@ -44,6 +39,16 @@ describe('Integration - Deployment Pagination', () => {
     E3 = await buildDeployData([P3], { type, timestamp: timestamp + 1, metadata: { a: 'metadata3' } })
   })
 
+  beforeEach(() => resetServer(server))
```

Comment:
and here

Reply:
I think this is ok, it returns a promise, so it will be awaited